lib/mqtt_abstraction_impl.cpp

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+#include "utils/message_queue.hpp"
```

Comment:
Please try to keep the includes categorized.

Reply:
Unintended consequence of using an editor that tries to be helpful and adds include files for you (whether you want them or not).

Reply:
will remove along with any other unneeded includes